scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
No we should not stop.
This updating of the web console is done in conjunction with the upstream vendoring the web console.  The OCP stuff has to be put in at some time, and this is when.